performance_test/src/experiment_configuration/experiment_configuration.hpp

```diff
@@ -150,18 +176,35 @@ class ExperimentConfiguration
     m_roundtrip_mode(RoundTripMode::NONE)
   {}
 
+#ifdef ODB_FOR_SQL_ENABLED
+  friend class odb::access;
+#endif
+
   /// Throws #std::runtime_error if the experiment is not set up.
   void check_setup() const;
 
   /// Generates filename from the experiment configuration and opens a file accordingly. This will
   /// throw if the experiment configuration is not set up.
   void open_file();
 
+  // Using the GUID of the experiment as ID.
+#ifdef ODB_FOR_SQL_ENABLED
+  #pragma db id
```

Comment:
Can include a usage reference for these https://www.codesynthesis.com/products/odb/doc/manual.xhtml#6.1  or if you find some other section more relevant.

Reply:
I don't understand what you mean here...

Reply:
I was saying something like this : https://github.com/ApexAI/performance_test/pull/84/files/cd7fb38dae04082eb2ec03ddef3e42905b138b75#diff-d227b9c326138e7ff6d0501f7c979654R92

Means if someone else reads our code then they can get a reference that we referred odb manual for this.

Reply:
I added a note in readme that you should refer to odb manual for more implementation details, have a look, I think it's cleaner to add it in readme than in the code.

Reply:
Okay sounds good!